src/actinia_core/core/stac_importer_interface.py

```diff
@@ -142,26 +144,70 @@ def _stac_import(self, stac_collection_id=None, semantic_label=None,
 
             stac_processes = []
 
+            # Create the strds
+
+            # datetime object containing current date and time
+            now = datetime.now()
+
+            # dd/mm/YY H:M:S
+            dt_string = now.strftime("%d%m%Y%H%M%S")
+
+            t_name = f'{stac_name}{dt_string}'
+
+            exec_params = ["type=strds",
+                           "temporaltype=absolute",
+                           "output=%s" % t_name,
+                           "title=%s" % stac_name,
+                           "description=%s" % stac_collection_id
+                           ]
+
+            p = Process(
+                exec_type="grass",
+                executable="t.create",
+                executable_params=exec_params,
+                id=f"t_create_{os.path.basename(stac_name)}",
+                skip_permission_check=True
+            )
+
+            stac_processes.append(p)
+
             for key, value in stac_result.items():
 
-                for name_id, url in value.items():
+                for name_id, url, item_date in value.items():
 
                     output_name = stac_name + "_" + key + "_" + name_id
 
                     # From Here Onwards, the Process build starts
                     exec_params = ["input=%s" % "/vsicurl/"+url,
                                    "output=%s" % output_name,
-                                   "-o"]
+                                   "extent=region",
+                                   "resolution=region"]
 
                     p = Process(
                         exec_type="grass",
-                        executable="r.in.gdal",
+                        executable="r.import",
                         executable_params=exec_params,
-                        id=f"r_gdal_{os.path.basename(output_name)}",
+                        id=f"r_import_{os.path.basename(output_name)}",
                         skip_permission_check=True
                     )
 
```

Comment:
After import, you can set the semantic label with `r.support`. `t.register` will then automatically use this semantic label and insert the raster map with metadata accordingly in the TGIS db.

Reply:
is it [`r.support`](https://grass.osgeo.org/grass78/manuals/r.support.html)? what's the difference with [`r.semantic.label`](https://grass.osgeo.org/grass80/manuals/r.semantic.label.html)

Reply:
It needs to be v8x: https://grass.osgeo.org/grass80/manuals/r.support.html#set-semantic-label 
Which of the two is preferred, I cannot say.